contents/docs/tutorials/nuxt-js-integration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Tracking Nuxt.js apps
+sidebar: Docs
+showTitle: true
+---
+
+If you are using Nuxt.js and want to track your application using PostHog this tutorial might help you out. 
+
+It will guide you through an example integration of PostHog using Nuxt.js plugins. 
+
+### Is This Tutorial For Me?
+
+This tutorial is aimed at Nuxt.js users which run Nuxt in `spa` or `universal` mode. 
+We are going to look at some minimal example code which should get you started quickly and provide a base for further customization.
+
+### Pre-Requisites
+
+To follow this tutorial along, you need to:
+
+1. Have [deployed PostHog](/docs/deployment).
+2. Have a running Nuxt.js application
+
+### Minmal Example
```

Comment:
Minmal -> Minimal 

Unfortunately the spell checker bot doesn't work on forks :(

Reply:
Thanks for pointing that out. Oversaw it